### PR TITLE
Add multi-reply tip

### PR DESCRIPTION
**Pull request by Vinnl**
# New feature description

This adds instructions on how to reply to multiple senders.

# Screenshot (if applicable)

[See preview site.](https://deploy-preview-2781--fx-relay-demo.netlify.app/accounts/profile/?mockId=some)

# How to test

Open the email masking dashboard (adding it to the phone masking dashboard is in https://github.com/mozilla/fx-private-relay/pull/2780) with an account that has phone masking setup. You should see the multi-reply tip. You should not see it with a user that does not have a Relay number yet.

# Checklist

- [x] l10n changes have been submitted to the l10n repository, if any. https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/114
- [x] All acceptance criteria are met.
- [x] I've added or updated relevant docs in the docs/ directory.
- [x] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/frontend/src/styles/tokens.scss`).
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

**Comment by Vinnl** [comment]
Added e1fc191 to add a still image alternative for people who either don't have the right codecs or have disabled animations.

**Comment by Vinnl** [comment]
And also added an alt text in 0e1a56fd.

File: frontend/src/hooks/api/relayNumber.ts [hunk 1]
```diff
@@ -27,16 +27,29 @@ export type UpdateForwardingToPhone = (
...
+
+export type UseRelayNumberOptions = Partial<{
+  /**
+   * Set to `true` to prevent requests to the /relaynumber/ API
+   *
+   * This is useful if e.g. you already know that the user does not have a phone
+   * masking subscription.
+   */
+  disable?: boolean;
+}>;
```
Comment (lloan) [inline frontend/src/hooks/api/relayNumber.ts:39]: This seems useful!

**Review by lloan — APPROVED** [review]
Code looks good to me. I can see this working as intended:  
image

**Comment by Vinnl** [comment]
Thanks! For full disclosure, I also added 95415fc, because [the a11y team suggested](https://mozilla.slack.com/archives/C4E0W8B8E/p1668464083290679?thread_ts=1668160957.194549&cid=C4E0W8B8E) that a `title` on the video might be read by screen readers, and it's useful anyway.

**Pull request merged**